api.planx.uk/modules/webhooks/docs.yaml

```diff
@@ -66,53 +63,47 @@ components:
     EmailSubmission:
       type: object
       properties:
-        body:
+        event:
```

Comment:
Previously, these were incorrectly wrapped in `body` which means the example payload in the API docs are incorrect.

![image](https://github.com/user-attachments/assets/69c51d0a-c7bf-4b36-b9ff-ad1c65b50bec)